srcDoc/release_notes.rst

```diff
@@ -51,6 +51,12 @@ Release Notes Master
 What's New in Master
 ~~~~~~~~~~~~~~~~~~~~
 
+Build HTML and PDF documentation (#94)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now from the `build.sh` script, *RTI Perftest* can generte the HTML and PDF
+documentation.
+
```

Comment:
*RTI Perftest* build script for linux now offers the option to generate the HTML and PDF documentation from the `rst` files in `srcDoc`.

Reply:
Done